### DISCUSSION TITLE
White screen

### DISCUSSION THREAD

**Discussion by OtakuGuy** [agalwood/Motrix#1626] [Q&A]
I was hopping if someone could help me solve this issue. So the app will turn white, and I assume that the download has stopped, and is very annoying. Does anyone know a solution to this problem?

**Comment by RomeroNX** [comment]
I have the same problem. You just need to move the folder from Motrix to C drive to any location

**Reply by dragonunitedtm** [reply]
did that for another reason , didnt work.

**Comment by williamvyong** [comment]
u mean the whole motrix folder ? or the download folder ?
